app/controllers/advanced_messenger_controller.rb

```diff
@@ -148,14 +145,4 @@ def ignore_all_unread_entities(unread_entities, is_visible)
       format.js {render inline: "location.reload();" }
     end
   end
-
-  # private helper method
-  def is_journal_visible (journal)
```

Comment:
Aceasta metoda mai era folosita in acest controller astfel: 
![image](https://github.com/user-attachments/assets/c29e0d03-d79c-4bea-a057-0036eb4f6fba)

Cred ca mutand-o in helper, nu mai este accesibila cu `:`

Cum acest caz este mai greu de testat pune te rog un log (cu `Rails.logger.debug`) in metoda de mai sus. Suspectez ca `:is_journal_visible` este `nil`. Daca da de vazut cum obtii referinta. Probabil direct cu `is_journal_visible`

Idem si pentru cealalta metoda

Reply:
ok. E  bine